### PR TITLE
Updating ilxutils

### PR DESCRIPTION
**Pull request by tmsincomb**
I fetched and merged pyontutils contents to last pushed and updated ilxutils contents.

### CODE REVIEW

**Comment by tmsincomb** [comment]
Deleted all local files and replaced them with the downloaded source.

**Comment by tmsincomb** [comment]
Added descriptions for the docopts.

**Comment by tmsincomb** [comment]
removed the .DS_store

File: ilxutils/ilxutils/api_classes_wrappe.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import json
```
Comment (tgbugs) [inline ilxutils/ilxutils/api_classes_wrappe.py]: @tmsincomb Is this file used anywhere? It looks like the filename is misspelled.

File: ilxutils/ilxutils/elastic_commands.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+docker run --rm -ti -v  /home/troy/elastic_testing:/elastic_testing taskrabbit/elasticsearch-dump \
```
Comment (tgbugs) [inline ilxutils/ilxutils/elastic_commands.py]: I think this should be named elastic_commands.sh?

File: ilxutils/ilxutils/generic_existing_ids_upload.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        if new_term_data[key] != old_term_data[key]:
+            print('FAILED')
+            print(key)
+            print(new_term_data[key])
+            print(old_term_data[key])
+            sys.exit(str(k)+'\n'+str(old_term_data)+'\n'+str(new_term_data))
+
+#subj_to_tid = get_subj_to_tid(engine_key)
+filename = 'OEN-NLXOEN'
+new_existing_ids = json.load(open('/Users/tmsincomb/Desktop/dump/Updating_Prefixes/'+filename+'.json'))
```
Comment (tgbugs) [inline ilxutils/ilxutils/generic_existing_ids_upload.py]: Can you please find and switch all direct uses of `open` to `with open` so we don't have to worry about the number of dangling file descriptors (pypy has some issues with those).

File: ilxutils/ilxutils/get_ttl_from_single_term.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+class Single_Term_Turtle_Generator():
+
+    def __init__(self):
+        self.args = self.read_doc(docopt(__doc__, version=VERSION))
+        self.p = pathlib.PurePath(self.args.output)
+        self.g = self.create_graph()
+        self.pref_dict, self.ilx_to_pref, self.unpref_dict =  self.make_preferred_iris_dict()
+
+    def myopen(self, path):
+        return open(path, 'r').read().strip()
```
Comment (tgbugs) [inline ilxutils/ilxutils/get_ttl_from_single_term.py]: another -> with open example

File: ilxutils/ilxutils/interlex2ttl.py [hunk 5]
```diff
@@ -0,0 +1,338 @@
+#!/usr/local/Cellar/python3/3.6.1/bin/python3.6
```
Comment (tgbugs) [inline ilxutils/ilxutils/interlex2ttl.py]: all of these shebangs need to be converted to `#!/usr/bin/env python3.6`

File: pyontutils/core.py [hunk 6]
```diff
@@ -569,6 +569,7 @@ def parse(self, *triples, root=None, graph=None):  # drop, parse, contract
             [graph.add(t) for t in triples]
 
         self.triples = []
+        failed_triples = []
```
Comment (tgbugs) [inline pyontutils/core.py]: Can you remove the changes to this file please?

**Pull request merged**